faf-commons-data/src/main/java/com/faforever/commons/replay/ReplayDataParser.java

```diff
@@ -379,21 +379,36 @@ private void parseGiveResourcesToPlayer(Map<String, Object> lua) {
 
 
   void parseModeratorEvent(Map<String, Object> lua, Integer player) {
-    String messageContent = "Content of Message Missing";
-    int fromInt = -1; // Default Value
+    String messageContent = null;
+    String playerNameFromArmy = null;
+    String playerNameFromCommandSource = null;
     int activeCommandSource = -1; // Default Value
+    int fromArmy = -1; // Default Value
```

Comment:
What is -1 actually meaning? civilian?

Reply:
Observers have that value, it should probably default to -2